src/components/Adds/Adds.js

```diff
@@ -29,96 +30,108 @@ export default function Adds() {
-          required
-        />
-        <TextField
-          id="outlined-multiline-static"
-          label="Description"
-          multiline
-          rows={4}
-          type="text"
-          name="description"
-          placeholder="Description"
-          required
-        />
-        <Button type="submit" variant="contained" color="primary">Submit</Button>
+        <div className="input">
+          <TextField
+            id="standard-basic"
+            label="Model"
+            variant="outlined"
+            type="text"
+            name="model"
+            placeholder="Model"
+            required
+          />
+          <TextField
+            id="standard-basic"
+            label="Color"
+            variant="outlined"
+            type="text"
+            name="color"
+            placeholder="Color"
+            required
+          />
+        </div>
+        <div className="input">
+          <TextField
+            id="standard-basic"
+            label="Image"
+            variant="outlined"
+            type="text"
+            name="image"
+            placeholder="https://"
+            required
+          />
+          <TextField
+            id="standard-basic"
+            label="Range"
+            variant="outlined"
+            type="integer"
+            name="range"
+            placeholder="Range"
+            required
+          />
+        </div>
+        <div className="input">
+          <TextField
+            id="standard-basic"
+            label="Motor type"
+            variant="outlined"
+            type="text"
+            name="motor_type"
+            placeholder="Motor type"
+            required
+          />
+          <TextField
+            id="standard-basic"
+            label="Acceleration time"
+            variant="outlined"
+            type="decimal"
+            name="acceleration_time"
+            placeholder="Acceleration time"
+            required
+          />
+        </div>
+        <div className="input">
+          <TextField
+            id="standard-basic"
+            label="Booking price"
+            variant="outlined"
+            type="decimal"
+            name="booking_price"
+            placeholder="Booking price"
+            required
+          />
+          <TextField
+            id="standard-basic"
+            label="Booking duration"
+            variant="outlined"
+            type="integer"
+            name="booking_duration"
+            placeholder="Booking duration"
+            required
+          />
+        </div>
+        <div>
+          <TextField
+            id="outlined-multiline-static"
+            label="Description"
+            multiline
+            rows={4}
+            type="text"
+            name="description"
+            placeholder="Description"
+            required
+          />
+        </div>
+        <div>
+          <Button type="submit" variant="contained" color="primary">Submit</Button>
+        </div>
```

Comment:
[OPTIONAL] Kindly consider changing the button color to stick with the design pallet.

Reply:
Could we do it in the final style revision, please?